<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressDownloadDialog</class>
 <widget class="QDialog" name="AddressDownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddressDownloadDialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="addressmanagement.qrc">
    <normaloff>:/img/download.png</normaloff>:/img/download.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>List Select</string>
   </property>
   <widget class="QCheckBox" name="checkBox_All">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_PhoneNumber">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PhoneNumber</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Group">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Memo">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Memo</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Download">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/download.png</normaloff>:/img/download.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_Group">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>210</width>
     <height>240</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Group Select</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_OutputGroup">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>210</width>
     <height>240</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/rightarrow.png</normaloff>:/img/rightarrow.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="addressmanagement.qrc">
     <normaloff>:/img/leftarrow.png</normaloff>:/img/leftarrow.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="addressmanagement.qrc"/>
 </resources>
 <connections/>
</ui>
